lib/collectinfo_analyzer/collectinfo_handler/log_handler.py

```diff
@@ -151,8 +153,15 @@ def get_ip_to_node_id_mapping(self, timestamp):
 
         return ip_to_node
 
-    def info_getconfig(self, stanza="", flip=False):
-        return self._fetch_from_cinfo_log(type="config", stanza=stanza, flip=flip)
+    def info_getconfig(
+        self,
+        stanza="",
+        flip=False,
+        nodes: NodeSelectionType = NodeSelection.ALL,
```

Comment:
added ability to select 'principal' node

Reply:
In the future, we might support selecting a list of specific nodes like in live_cluster mode. Currently, we just support "all" (default) or 'principal"